samples/host_exerciser/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ opae_add_executable(TARGET host_exerciser
         opae-c
         opae-cxx-core
         afu-test
+        cap
```

Comment:
Before we can link with this new library, we need to implement a finder.  Please see opae-sdk/opae-libs/cmake/modules/Find*